Orange/widgets/utils/colorpalette.py

```diff
@@ -597,66 +597,52 @@ def __getitem__(self, val):
 
 
 class ColorPaletteGenerator:
-    maxHueVal = 260
 
-    def __init__(self, number_of_colors=None, rgb_colors=DefaultRGBColors):
+    def __init__(self, number_of_colors=0, colors=DefaultRGBColors):
```

Comment:
I am very much in favour of improving the code beyond the minimum required changes in general. If possible/applicable the change and other improvements are better in separate commits.
Just test both, the change and other improvements :)
In our cases the author usually knows the implications of his pull request better than the one merging it, so don't count on somebody else to find errors.

The patch looks OK at first glance, finish with testing and changes if any more are planned and I'll take a closer look.